# g:Profiler



**g:GOSt** performs functional enrichment analysis, also known as over-representation analysis (ORA) or gene set enrichment analysis, on input gene list. It maps genes to known functional information sources and detects statistically significantly enriched terms. We regularly retrieve data from [Ensembl database](https://www.ensembl.org/) and fungi, plants or metazoa specific versions of [Ensembl Genomes](http://ensemblgenomes.org/), and parasite specific data from [WormBase ParaSite](https://parasite.wormbase.org/). In addition to [Gene Ontology](http://geneontology.org/), we include pathways from [KEGG](https://www.genome.jp/kegg/) [Reactome](https://reactome.org/) and [WikiPathways](https://www.wikipathways.org/); miRNA targets from [miRTarBase](http://mirtarbase.mbc.nctu.edu.tw/) and regulatory motif matches from [TRANSFAC](http://genexplain.com/transfac/); tissue specificity from [Human Protein Atlas](https://www.proteinatlas.org/); protein complexes from [CORUM](http://mips.helmholtz-muenchen.de/corum/) and human disease phenotypes from [Human Phenotype Ontology](https://hpo.jax.org/). g:GOSt supports close to 500 organisms and accepts hundreds of identifier types.



{% embed url="https://biit.cs.ut.ee/gprofiler/gost" %}
[https://biit.cs.ut.ee/gprofiler/gost](https://biit.cs.ut.ee/gprofiler/gost)
{% endembed %}
